docs/Tutorials/Create-Private-Network.md

```diff
@@ -21,8 +21,8 @@ To create a private network:
 
 1. [Create Folders](#1-create-folders)
 1. [Create Genesis File](#2-create-genesis-file)
-1. [Start First Node and Get Node Public Key](#3-start-first-node-and-get-node-public-key)
-1. [Restart First Node as Bootnode](#4-restart-first-node-as-bootnode)
+1. [Get Node Public Key of First Node](#3-start-first-node-and-get-node-public-key)
```

Comment:
Could this step say "Get Public Key of First Node"?

Reply:
Indeed yes, it doesn't start the node, only exports the key
Also if **3. Start First Node and Get Node Public Key ** is renamed to **3. Get Node Public Key of First Node**, the link will change too, so need to be `#3-get-node-public-key-of-first-node`

Reply:
Done

Reply:
Done